### ISSUE TITLE
[Sample]    d_loss :           g_loss : 

### ISSUE THREAD

**Issue by MPLAP** [carpedm20/DCGAN-tensorflow#362]
I trained 100 pictures and saved sample in units of 1000.
Whenever a sample is generated, the following text appears.

[Sample]        d_loss :            g_loss :

But in most cases d_loss is close to zero, In the case of g_loss, the output is in units of tens or hundreds.

Can you explain in detail about the loss value like this?

**Comment by MPLAP** [comment]
one more!
How do I display the graph in Training details? Thanks :D
